src/aat/resources/features/F-056/S-582.td.json

```diff
@@ -0,0 +1,159 @@
+{
+  "title": "must update successfully the respective fields with ACL permissions for a Citizen",
+  "_extends_": "S-582_Update_Case_For_Citizen_Data_Base",
+  "_guid_": "S-582",
+  "specs": [
+    "an active citizen profile in CCD with update permissions for certain fields but not for others in a given case type",
+    "is made to update the document metadata in DocumentField2, which the user has update permissions for",
+    "contains updated values for DocumentField2"
+  ],
+  "user": {
+    "username": "befta.citizen.2@gmail.com",
+    "password": "Pa55word11"
+  },
+  "request": {
+    "body": {
+      "data": {
+        "DocumentField2":{
+          "document_url":"http://dm-store-aat.service.core-compute-aat.internal/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943",
+          "document_binary_url":"http://dm-store-aat.service.core-compute-aat.internal/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943/binary",
+          "document_filename":"Elastic Search test Case.png --> updated by Citizen"
+        }
+      }
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 201,
+    "responseMessage": "OK",
+    "headers": {
+      "_extends_": "Common_Response_Headers",
+      "Content-Encoding" : "gzip",
+      "vary" : "accept-encoding"
+    },
+    "body" : {
+      "id": "[[ANY]]",
+      "jurisdiction": "BEFTA_JURISDICTION_2",
+      "state": "TODO",
+      "version": "[[ANY]]",
+      "case_type_id": "BEFTA_CASETYPE_2_1",
+      "created_date": "[[ANY]]",
+      "last_modified": "[[ANY]]",
+      "security_classification": "PUBLIC",
+      "case_data": {
```

Comment:
please use the common inheritance feature if possible

Reply:
Done...please hav a look